k8sfv/k8sfv.go

```diff
@@ -211,6 +211,7 @@ func cleanupAll(clientset *kubernetes.Clientset, nsPrefix string) {
 
 func panicIfError(err error) {
 	if err != nil {
+		log.WithError(err).Error("About to panic...")
```

Comment:
Is it useful to keep this?  (Just checking.)

Reply:
Might be useful in case the panic gets swallowed and logged out-of-order by Ginkgo?